taxonomies/categories.txt

```diff
@@ -65094,6 +65097,10 @@ en:Garam masala
 ja:ガラムマサラ
 xx:Garam masala
 
+<en:Spices
```

Comment:
```suggestion
<en:Flavour enhancers
```

Reply:
@OpenChris additives.txt is automatically included in ingredients.txt, so glutamate should already be covered, did you find products where it doesn't work?

Reply:
ah my mistake I thought it was ingredients.txt and not categories.txt